gocourse02/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	zoo "gocourse02/zoo"
+)
+
+func main() {
+	lion := zoo.NewLion(zoo.NewAnimal("Mykola", zoo.Male), "green")
+	lioness := zoo.NewLion(zoo.NewAnimal("Natalia", zoo.Female), "white")
```

Comment:
Чи можемо ми користувачу спростити життя?

Переробити трішки методи, щоб працювало ось так:
```suggestion
	lion := zoo.NewLion("Mykola", zoo.Male, "green")
	lioness := zoo.NewLion("Natalia", zoo.Female, "white")
```